resqpy/organize/organize_functions.py

```diff
@@ -3,6 +3,7 @@
 version = '9th August 2021'
```

Comment:
This docstring is out of date. Moreover - propose changing the name of this submodule to `_utils.py` or `_common.py`

- Functions seem to be for internal use only, so an underscore to remove from public api
- Name is currently `resqpy.organize.organize_functions` , so can remove redundant repetition of "organize". Module names should generally be short, ideally one word

Reply:
changed it to _uitils.py.